corehq/apps/app_manager/templates/app_manager/partials/modules/case_list.html

```diff
@@ -143,6 +143,79 @@ <h4 class="panel-title panel-title-nolink">
           </label>
         </div>
       </div>
+
+      <span> This preview just shows the relative position of fields and is not an accurate representation.</span>
```

Comment:
This is unfortunate. Is it not possible to more accurately match the layout?

Reply:
@orangejenny That was based on your comment `<!-- TODO: some kind of warning that this isn't pixel by pixel accurate -->`. I can remove it.

Reply:
(noting this is removed now)